src/main/java/athena/timetable/Timetable.java

```diff
@@ -21,8 +23,8 @@ public Timetable(TaskList taskList) {
         populateTimetable();
     }
 
-    public Timetable(TaskList taskList, TaskFilter taskFilter) {
-        this.taskList = taskList.getFilteredList(taskFilter);
+    public Timetable(TaskList taskList, TaskFilter taskFilter, ForecastFilter forecastFilter) {
+        this.taskList = taskList.getFilteredList(taskFilter, forecastFilter);
```

Comment:
Continuing the comment above about `getFilteredList(TaskFilter taskFilter)`, we can do `getFilteredList(importanceFilter).getFilteredList(forecastFilter)`.